src/components/Route/CreateRoute.tsx

```diff
@@ -0,0 +1,195 @@
+      <FormControl isInvalid={formError} p={1}>
+        <VStack space={1}>
+          <Text fontSize='lg' alignSelf='center'>Create a new Route</Text>
+          <FormControl.Label isRequired>Route name</FormControl.Label>
+          <Input isRequired type='text' onChangeText={setName} placeholder='Name'/>
+          <FormControl.Label>Route Thumbnail</FormControl.Label>
+          <input type='file' accept='image/*' onChange={handleFileSelect}/>
+          <FormControl.Label isRequired>Route type</FormControl.Label>
+          <Select selectedValue={type} accessibilityLabel='choose route type'
+            placeholder='Route type' onValueChange={ (value) => {
+              const route_type = value as RouteType;
+              setRawgrade(undefined);
+              setType(route_type);
+            }}>
+            {Object.values(RouteType).map( (routetype) => {
+              return <Select.Item key={routetype} label={routetype} value={routetype}/>;
+            })}
+          </Select>
+          <FormControl.Label isRequired>Route grade</FormControl.Label>
+          <Select isDisabled={type === undefined} placeholder='Route grade'
+            onValueChange={ (value) => setRawgrade(parseInt(value))}>
+            {type !== undefined &&
+              RouteTypeToGetAllClassifiers(type).map( (value) => {
+                return <Select.Item value={''+value.rawgrade} label={value.displayString}
+                  key={value.displayString}/>;
+              })
+            }
+          </Select>
+          <FormControl.Label isRequired>Rope</FormControl.Label>
+          <Select placeholder='Rope' onValueChange={ (ropeString) => setRope(parseInt(ropeString))}>
+            {Ropes.map( (val) => {
+              return <Select.Item key={val} label={val} value={val}/>;
+            })}
+          </Select>
+          <FormControl.Label isRequired>Color</FormControl.Label>
+          <Select placeholder='Color' onValueChange={ (colorString) => {
+            const color = colorString as RouteColor;
+            setRouteColor(color);
+          }}>
+            {Object.values(RouteColor).map( (color) => {
+              return <Select.Item key={color} label={color} value={color}/>;
+            })}
+          </Select>
+          <FormControl.Label isRequired>Natural Rules</FormControl.Label>
+          <Select placeholder='Natural Rules' onValueChange={ (rulesString) => {
+            const rules = rulesString as RouteTech;
+            setRules(rules);
+          }}>
+            {Object.values(RouteTech).map( (rules) => {
+              return <Select.Item key={rules} label={rules} value={rules}/>;
+            })}
+          </Select>
+          <FormControl.Label>Setter</FormControl.Label>
+          {overrideSetterBool ?
+            <Input isRequired={false} type='text' onChangeText={setOverrideSetterString}
+              placeholder='Setter (optional)'/>
+            :
+            <Text>This will be a user search component</Text>
```

Comment:
bet, TODO/extra issue for putting this into this component when UserSearch is ready?

Reply:
Just made the issue